useful/template.secrets.py

```diff
@@ -19,3 +19,10 @@
 ssh-keygen -o #Doesn't work
 ssh-keygen -t rsa -b 4096
 """
+
+# Discord
+DISCORD = {
+    "URL": {
+        "WELCOME": "https://discord.gg/44yUVeU"
```

Comment:
This should probably be named `DISCORD_INVITE_LINK` instead of `WELCOME`, for clarity.

Reply:
I agree, was just an easy name to represent it as the welcome link